pkgs/development/python-modules/pyqtwebengine/default.nix

```diff
@@ -0,0 +1,55 @@
+{ lib, fetchurl, pythonPackages, pkgconfig
+, qmake, qtbase, qtsvg, qtwebengine
+}:
+
+let
+
+  inherit (pythonPackages) buildPythonPackage python isPy3k pyqt5 enum34;
+
+  sip = pythonPackages.sip.override { sip-module = "PyQt5.sip"; };
+
+in buildPythonPackage rec {
+  pname = "pyqtwebengine";
+  version = "5.12";
+  format = "other";
+
+  src = fetchurl {
+    url = "https://www.riverbankcomputing.com/static/Downloads/PyQtWebEngine/PyQtWebEngine_gpl-${version}.tar.gz";
```

Comment:
```
nix-prefetch-url "https://www.riverbankcomputing.com/static/Downloads/PyQtWebEngine/PyQtWebEngine_gpl-5.12.tar.gz"        
[0.0 MiB DL]
error: unable to download 'https://www.riverbankcomputing.com/static/Downloads/PyQtWebEngine/PyQtWebEngine_gpl-5.12.tar.gz': HTTP error 404
```
```suggestion
    url = "https://www.riverbankcomputing.com/static/Downloads/PyQtWebEngine/${version}/PyQtWebEngine_gpl-${version}.tar.gz";
```

Reply:
That, and version 5.12 doesn't exist anymore — you need 5.12.1

Reply:
It does:
```
nix-prefetch-url https://www.riverbankcomputing.com/static/Downloads/PyQtWebEngine/5.12/PyQtWebEngine_gpl-5.12.tar.gz
[0.0 MiB DL]
path is '/nix/store/8wdwbx0vir0jkyfv4cnry1x1p1pd4625-PyQtWebEngine_gpl-5.12.tar.gz'
0j9zzgwrvh61mrzjfcdlhr08vg931ycb53ri51vynsj0grp07smn
```

Reply:
Hah, the lack of version in path, compounded by the availability of the new version, tripped me up.